### PR TITLE
[SYSSETUP] Add marquee progress bar to show setup is alive

### PR DESCRIPTION
**Pull request by binarymaster**
Add marquee progress bar to show the setup is still alive and not frozen.

JIRA issue: [CORE-14163](https://jira.reactos.org/browse/CORE-14163)

GIF demo: https://gfycat.com/ExaltedNaturalAustrianpinscher

### CODE REVIEW

File: dll/win32/syssetup/install.c [hunk 1]
```diff
@@ -567,6 +567,9 @@ ShowStatusMessageThread(
 
     ShowWindow(hWnd, SW_SHOW);
 
+    hItem = GetDlgItem(hWnd, IDC_STATUSPROGRESS);
+    if (hItem) PostMessage(hItem, PBM_SETMARQUEE, TRUE, 40);
+
```
Comment (Fraizeraust) [inline dll/win32/syssetup/install.c]: Is the 571 line supposed to be this way? Shouldn't the conditional code be wrapped with curly braces? Like in this image:
https://imgur.com/a/KuWmp

Comment (binarymaster) [reply dll/win32/syssetup/install.c]: Don't know whether it's required here or not, but since there is only one operation, it works.

**Comment by HBelusca** [comment]
I like it!!

File: dll/win32/syssetup/install.c (hunk 1 above)
Comment (ThFabba) [reply dll/win32/syssetup/install.c]: This works, although it's preferred to use a separate line (for step-through) and to always use braces.

Comment (binarymaster) [reply dll/win32/syssetup/install.c]: Allright then :)

**Comment by ColinFinck** [comment]
Either this, or we could keep consistent with the new boot screen, winlogon, and winver image by turning the "Installing devices" dialog into something like http://www.windows2000.windowsreinstall.com/installnewhdd/Image48.jpg
In this image, Windows shows activity by moving the blue bar in a marquee-like way.

**Comment by binarymaster** [comment]
> by turning the "Installing devices" dialog into something like

This would require some GINA magic... (and yep, I also thought about that) but our GINA lacks rotation line for now.

**Comment by HBelusca** [comment]
You both have good points, however as @binarymaster mentioned the marquee stuff is in the winlogon GINA, and this is completely dissociated from the syssetup code. And, since the syssetup "Installing drivers..." is shown only once, basically when starting 2nd-stage setup (but then the GINA does not display its message boxes),we should not really care whether or not the "Installing drivers" dialog has the very same style as the GINA ones.

About GINA, it would indeed be really nice to have that too; I suppose the code to make the bar rotate should look "similar" to the one we used in the NTOS:INBV code, i.e. playing with BitBlt and a window timer.

**Comment by HBelusca** [comment]
So let's keep it as it is and commit. Doing the animation in the GINA will be for later!

**Pull request merged**